### DISCUSSION TITLE
higress gateway 作为UpStream时的连接数的问题

### DISCUSSION THREAD

**Discussion by hoocoo** [alibaba/higress#936] [Q&A]
默认连接数是多少呢？
该怎么调整连接数的大小？

**Comment by johnlanni** [comment]
只有有最大限制，没有默认连接数这一说法

**Reply by hoocoo** [reply]
最大是限制多少呢？

**Reply by hoocoo** [reply]
标题写错了，应该是作为upStream时

**Comment by johnlanni** [comment]
作为upstream是什么意思

**Comment by johnlanni** [comment]
如果你说的是higress跟upstream之间的连接配置的话，看下这个：
![image](https://github.com/alibaba/higress/assets/6763318/4e70eadb-e3fa-4493-b745-b3fa7b2d197d)
https://higress.io/zh-cn/docs/user/configurations

**Reply by hoocoo** [reply]
higress作为alb的upstream，alb跟higress建立的连接数是否有最大限制？

**Reply by johnlanni** [reply]
higress 没有

**Reply by hoocoo** [reply]
![image](https://github.com/alibaba/higress/assets/31276333/c59e6889-559a-4d57-91a8-89f64df7a6f8)
![image](https://github.com/alibaba/higress/assets/31276333/3fdf4469-90b4-4f66-b7f9-f737a17ba077)
看抓包两次请求间隔大于180秒，DownStream 超时也是180秒，有没有可能是这样原因导致的502？

**Reply by hoocoo** [reply]
![image](https://github.com/alibaba/higress/assets/31276333/44745fca-4636-4457-b0b7-392bd7933b4d)
可以帮忙看看怎么调整idleTimeout这个参数吗？ 或者说我修改configMap后重启gateway，我怎么验证这个修改是否生效了？

**Reply by johnlanni** [reply]
修改后执行 hgctl gateway-config listener，搜索下对应配置项

**Reply by hoocoo** [reply]
![image](https://github.com/alibaba/higress/assets/31276333/48e4cd32-c5dd-4489-99b7-620d3c4a9905)
好的，已经生效了，谢谢！
